src/components/Molecules/List/FancyTweetList/FancyTweetList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  Typography,
+  CardActions,
+  Button,
+} from '@material-ui/core';
+
+interface IFancyTweetListProps {
+  tweets: any[];
+}
+
+const styles = {
+  card: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 140,
+  },
+};
+
+const FancyTweetList = ({ tweets }: IFancyTweetListProps) => {
+  if (tweets.length > 0) {
```

Comment:
I think it's a bit more readable when you use an early return. Something like;

```tsx
const FancyTweetList = ({ tweets }: IFancyTweetListProps) => {
  if (!tweets.length) {
    return null
  }

  return (
    <Card>
      ...
    </Card>
  )
}
```

Reply:
I removed the render condition anyway. It's checked one level up in `NextEventsBlock`.